ui-src/e2e/pages/configuration-PO.ts

```diff
@@ -91,8 +95,10 @@ class ConfigurationPageObject {
   setReportDir() {
     return this.reportsDir.sendKeys("./report");
   }
+ 
 
   addData() {
```

Comment:
Maybe this should be called setConfigData as that is what is happening in the function